interface-elements-for-web/articles/grid/data-summaries.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Data Summaries
+---
+# Data Summaries
+
+The grid can calculate and displays summaries, i.e., brief information about groups of rows or individual data columns. The following most popular aggregate functions are provided.
```

Comment:
"i.e., brief information"? Why 'brief information?

Reply:
i.e. -> that is. However, I do not understand why you need to use it here.

Reply:
I can remove it. 
"The grid can calculate and displays summaries about groups of rows or individual data columns."